docs/DeveloperGuide.adoc

```diff
@@ -721,20 +721,67 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |user |add multiple phone numbers for a contact |easily find an alternate number to contact someone
+
+|`* * *` | user | undo/redo my last action | recover from mistakes
 
 |`* * *` |user |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |use acronyms instead of the full command |use the app more efficiently
```

Comment:
Perhaps this can be just `* *` as using of acronyms is not that critical for users to use the application.